src/App.js

```diff
@@ -16,9 +16,12 @@ function App() {
           path="/"
           element={
             <ArticleList articles={articles} setArticles={setArticles} />
+            
           }
         />
         <Route path="/articles/:article_id" element={<SingleArticle />} />
+        <Route path="/articles/:article_id/comments" element={<SingleArticle />} />
```

Comment:
We shouldn't need an additional endpoint here. You're displaying the comments as part of the SingleArticle component regardless, so it's redundant to have both routes performing the same functionality.